dccrg.hpp

```diff
@@ -9748,6 +9754,12 @@ template <
 		this->cells_to_unrefine.clear();
 		this->cells_not_to_unrefine.clear();
 
+		for(const auto& cell : final_unrefines) {
```

Comment:
I think final_unrefines here will only contain local cells anyway, as the above is `for (const uint64_t unrefined: this->cells_to_unrefine) {` without any recursive looping or internal all-to-alls. So this loop is not needed after all.

Reply:
Removing it now.